### PR TITLE
Update connections.rst

### PR DESCRIPTION
**Pull request by dmcken**
Added documentation for how to disable ssl (an extra option) for this plugin which was not obvious to someone getting started.

### CODE REVIEW

File: docs/connections.rst [hunk 1]
```diff
@@ -3,3 +3,22 @@ Connections
...
+   :linenos:
+
+   labMT:
+     hostname: 172.16.1.1
+     platform: routeros
+     port: 8728
+     connection_options:
+       routerosapi:
+         extras:
+         use_ssl: False
```
Comment (devon-mar) [inline docs/connections.rst:24]:
```suggestion
           use_ssl: False
```

I believe that this should be indented

**Review by devon-mar — CHANGES_REQUESTED** [review]

**Comment by devon-mar** [comment]
Thanks for the PR @dmcken! I've left a comment for you to review.

Sorry about the poor documentation. I've been meaning to add some more detailed documentation and some more examples on usage but just haven't got around to writing it.

---

FYI, I'll be pushing a new release in a couple of days which fixes a couple of bugs and adds some tests.

File: docs/connections.rst (hunk 1 above)
Comment (dmcken) [reply docs/connections.rst:24]: Ahh, yes you are correct. My bad with missing that.

Its cool with the documentation, I can help as I'm now starting with nornir and can document from the perspective of someone new to nornir. The major thing that threw me off was the routerosapi module definition as everything else is routeros. Figured that out once I looked in plugins/connections/__init__.py .

**Comment by devon-mar** [comment]
Looks good! Could you rebase against devel?

**Comment by dmcken** [comment]
I am not having an easy time finding the rebase in github. I've done it down on my local machine and believe I've pushed it up. At this point I think it may be easier for me to create a new branch against develop and re-create a new pull request.

**Comment by dmcken** [comment]
So the rebase in Github desktop pulled in a bunch of old commits (24 total now).

If you are fine with it I'll close this one off and do a fresh pull request against devel.

**Comment by devon-mar** [comment]
The base branch was still `master` so that's why it was showing all those commits.

Thanks again for the PR!

**Pull request merged**